commands/shooter/aimshootertotarget.py

```diff
@@ -16,16 +17,48 @@ def __init__(self, shooter: ShooterSubsystem) -> None:
         self.shooter = shooter
         self.addRequirements([self.shooter])
 
+        self.timeInterp = Interpolator(constants.kTimeOfFlightList)
+
+    def distanceAngleOffsets(self) -> Tuple[number, Rotation2d]:
```

Comment:
Consider renaming to `calculateDistanceAndAngleOffsets` or `updateDistanceAndAngleOffsets` to reflect that this method does real work (calculations and interacting with `SmartDashboard`) rather than act as just a getter

Reply:
"`Distance`And`Angle`" (we are not calculating whatever a "DistanceAngle" is :-) )